deployability/modules/allocation/README.MD

```diff
@@ -0,0 +1,444 @@
+## Allocation Module
+
+### User documentation
+
+The Allocation module allows you to create and destroy VMs both locally and in AWS, it also gives the possibility of creating macOS VMs on Wazuh infrastructure or PowerPC VMs, as long as you have the necessary permissions to do so. VMs can be AMD64, ARM64, Windows, macOS (Intel and ARM), and PowerPC (Centos and Debian).
+
+#### Set the environment
+
+The execution of the allocation is carried out through the Workflow library, or by executing them manually through commands.
+Execution can be done from any operating system.
+
+Initially, You must install Python libraries. We recommend to use virtual environments. Follow the technical documentation at https://docs.python.org/3/library/venv.html.
+
+1. Activate the environment:
+
+  ```bash
+  source {venv directory}/bin/activate
+  ```
+
+2. Clone the `wazuh-qa` repository:
+
+  Navigate to the project directory and switch to the project branch:
+
+  ```bash
+  cd wazuh-qa
+  git checkout {project-branch}
+  ```
```

Comment:
Add a git clone of wazuh-qa using the HTTPS reference please

Reply:
Done a81aec813